### ISSUE TITLE
多GPU

### ISSUE THREAD

**Issue by yang-kuo** [HuangJunJie2017/BEVDet#338]
尊敬的作者您好：我想要在train.py文件里面修改参数来实现多gpu训练应该怎么改呢，我尝试了但没成功
![}PCDKH$LJ(5R QZJ6 XY5MD](https://github.com/HuangJunJie2017/BEVDet/assets/124995034/e1f58b9c-fd1d-4e38-9a9e-3605999f8bdc)
![12$KG9H~M}RVLM2~)L`LK5P](https://github.com/HuangJunJie2017/BEVDet/assets/124995034/15ec4dec-d8ad-4ae6-a2dc-7a4b3cb62e58)

**Comment by YuhanZhou521** [comment]
> 尊敬的作者您好：我想要在train.py文件里面修改参数来实现多gpu训练应该怎么改呢，我尝试了但没成功 ![}PCDKH$LJ(5R QZJ6 XY5MD](https://private-user-images.githubusercontent.com/124995034/321196533-e1f58b9c-fd1d-4e38-9a9e-3605999f8bdc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ1NjM5NzMsIm5iZiI6MTcxNDU2MzY3MywicGF0aCI6Ii8xMjQ5OTUwMzQvMzIxMTk2NTMzLWUxZjU4YjljLWZkMWQtNGUzOC05YTllLTM2MDU5OTlmOGJkYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwMVQxMTQxMTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zY2ExMTFhMTNiNGIxZmNhMDk5YTQ2YmUwODg5MjgxNWJmNGJmYmU3OTg0YTEzM2QwNjU2NjRiMjNkMzAzMjZiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.L9nJfqM7I5znvo3ZCDTFQej4ht9oEgg_Elkdp0QxR9o) ![12$KG9HM}RVLM2)L`LK5P](https://private-user-images.githubusercontent.com/124995034/321196574-15ec4dec-d8ad-4ae6-a2dc-7a4b3cb62e58.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ1NjM5NzMsIm5iZiI6MTcxNDU2MzY3MywicGF0aCI6Ii8xMjQ5OTUwMzQvMzIxMTk2NTc0LTE1ZWM0ZGVjLWQ4YWQtNGFlNi1hMmRjLTdhNGIzY2I2MmU1OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwMVQxMTQxMTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zNzI1ODMwYmNlNGY1YTI0ODhjOWEwMGQ2YmFkYTU3ZGM0ZjQ2NjZmY2NhYWU0MGI1M2VhMmNmMzJhNDNjMGM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ecVuP8C-MrwpThjeqGyDkZscQOFPaPnFrIcd_Axlnxs)

请问你解决了吗？

**Comment by L1bertad** [comment]
可以使用这个命令实现`CUDA_VISIBLE_DEVICES="1" python -m torch.distributed.launch --nnodes 1  --node_rank 0 --master_addr "127.0.0.1"  --nproc_per_node 1 --master_port 29500 tools/train.py configs/xxx --seed 0 --launcher pytorch
`
其中CUDA_VISIBLE_DEVICES设置为对应的gpu-id，nproc_per_node 设置为对应gpu数量
